go.mod

```
@@ -1,23 +1,43 @@
 module github.com/harvester/go-common
 
-go 1.23
+go 1.24
```

Without specifying a specific version, i'll get this error warning.
```
$ go get -u all
go: downloading go1.24 (linux/amd64)
go: download go1.24 for linux/amd64: toolchain not available
```
Goland IDE also likes it more to have a semvar with patch version.
```suggestion
go 1.24.1
```

---

updated!